### PR TITLE
[pota-value-test] Support comparison of scalar weights

### PR DESCRIPTION
**Pull request by jinevening**
This supports comparison of scalar weights.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

**Comment by jinevening** [comment]
The previous code failed when weights of an Op is a scalar (e.g., SPLIT).

This change enables to fetch the value of a scalar dataset in the h5 file.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]
LGTM

File: compiler/pota-quantization-value-test/compare_tensors.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def compare_quantization(tensor, tensor_name, expect_dir):
     for key in json_load:
         if key == "weights":
             expected_weights = np.array(json_load["weights"])
-            input_weights = tensor["weights"][:]
```
Comment (llFreetimell) [inline compiler/pota-quantization-value-test/compare_tensors.py:71]: Just curious, where can I find this grammar reference?

Comment (jinevening) [reply compiler/pota-quantization-value-test/compare_tensors.py:71]: You can find it here https://docs.h5py.org/en/stable/high/dataset.html#reading-writing-data.

Comment (mhs4670go) [reply compiler/pota-quantization-value-test/compare_tensors.py:71]: @llFreetimell https://docs.h5py.org/en/stable/high/dataset.html#reading-writing-data

**Pull request merged**